spec/imsc-hrm.html

```diff
@@ -237,9 +237,154 @@ <h2>Terms and Definitions</h2>
+  
+        <li>a first party over-provisions their <a>presentation processor</a> in
+        order to ensure a complete presentation of all <a>IMSC Document
+        Instance</a>, resulting in additional complexity or fewer resources
+        available to other processes; or</li>
+  
+        <li>one <a>presentation processor</a> becomes the de-facto reference to
+        determine whether the presentation of a <a>IMSC Document Instance</a>
+        will succeed, at the expense of other renderers and consistency of
+        presentation.</li>
+      </ul>
+
+      <figure id='fig-intro'>
+        <img src="figures/intro-fig-1.svg" alt="The HRM prevents incomplete presentations of IMSC Instance Documents.">
+
+        <figcaption>
+          The HRM allows authors and implementers of <a>presentation
+          processors</a> to agree on the maximum complexity of <a>IMSC Document
+          Instances</a>.
+        </figcaption>
+      </figure>
+
+      <p>As illustrated in <a href="#fig-intro"></a>, by defining a method (the
+      HRM) to compute the complexity of an <a>IMSC Document Instance</a> and
+      specifying a maximum <a>IMSC Document Instance</a> complexity according to
+      that method:</p>
+
+      <ul>
+        <li>authors can ensure that their <a>IMSC Document Instances</a> do not
+        exceed this maximum complexity;</li>
+  
+        <li>implementer can design their presentation processor can render all <a>IMSC
+        Document Instances</a> that do not exceed this maximum complexity.</li>
+      </ul>
+
+    </section>
+  
+    <section>
+      <h3>Why is the HRM needed to limit complexity?</h3>
+
+      <p>The HRM supplements the syntactic and structural constraints imposed in
+      [[IMSC]] by imposing constraints on the contents of the presentation.</p>
+  
+      <p>Because of the temporal and spatial variability of subtitles and
+        captions across types of content, territories and languages, it is not
+        possible to limit the complexity of an <a>IMSC Document Instance</a> using
+        only average values.</p>
+  
+      <p class="example">A constraint of 840 characters per minute would
+        allow a subtitle to contain 840 characters, as long it was the only one
+        within a 60 second window.</p>
+  
+      <p>The HRM achieves a more accurate representation of the complexity of an
+        <a>IMSC Document Instance</a> at any given time by taking into account
+        its past complexity in addition to its instantaneous complexity. The
+        same approach is commonly used in video to limit bitstream complexity,
+        e.g., the Hypothetical Reference Decoder (HRD) specified in
+        [[iso14496-10]].
+      </p>
+    </section>
+  
+    <section>
+      <h3>How does the HRM measures complexity?</h3>
+  
+      <p>
+        The HRM defines an (hypothetical) time required for painting subtitles
+        and captions. Painting includes drawing region backgrounds, rendering
```

Comment:
It is not clear to me what the meaning of a "hypothetical time" might be. 

```suggestion
        The HRM defines an estimate of the longest time required for painting subtitles
        and captions based on an hypothetical model of how a naive renderer might proceed.
        Painting includes drawing region backgrounds, rendering
```

Reply:
I do not like the term _naive_.  The third paragraph explains that the time is not intended to match the absolute time that an implementation will take to render the document, but rather scale with the absolute time that an implementation will take to render the document.

Reply:
See improved text.

Reply:
Yeah, I still don't think the phrase "hypothetical time" works well. First, it is not obvious what it means, second, it confuses the reader about the meaning of the Hypothetical Render Model - is it the model that's hypothetical, or the time? I've always understood it to be the model.

Also I think we are not quite doing everything the issue needs here, because we're not explaining that the purpose of the HRM is to define how an HRM validator needs to operate. How about this:

```suggestion
        The HRM defines how to compute the time that a hypothetical worst case
        presentation processor requires for painting subtitles and captions.
        Painting includes drawing region backgrounds, rendering and
```

Reply:
See improved text at https://github.com/w3c/imsc-hrm/commit/bdd49671c746f9ae3e99df5ef17952c96b0ce423